atlasdb-cassandra-integration-tests/build.gradle

```diff
@@ -14,4 +14,9 @@ dependencies {
 
   testCompile 'org.mockito:mockito-core:' + libVersions.mockito
   testCompile 'com.palantir.docker.compose:docker-compose-rule:' + libVersions.dockerComposeRule
+
+  test.doFirst {
+    println "Using cassandra version $cassVer"
+    environment "CASSANDRA_VERSION", "$cassVer"
+  }
```

Comment:
Let's do this within java. If we forget to copy this over, things are not going to be fun to debug.